pootle/apps/evernote_contact/forms.py

```diff
@@ -64,3 +65,26 @@ def from_email(self):
             self.cleaned_data['name'],
             self.cleaned_data['email']
         )
+
+
+class PootleReportForm(PootleContactForm):
+    """Contact form used to report errors on strings."""
+
+    report_email = forms.EmailField(
+        max_length=254,
+        required=False,
+        widget=forms.HiddenInput(),
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(PootleReportForm, self).__init__(*args, **kwargs)
+        self.fields.keyOrder += ['report_email']
```

Comment:
Can we add a note to mention this is necessary because `keyOrder` is altered in the parent form, please? Someone might think this line is unnecessary.